spec/features/user_logs_in_with_oauth_spec.rb

```diff
@@ -8,6 +8,6 @@
 
     click_link "Register with Yammer"
 
-    expect(page).to have_content("Welcome: foo")
+    expect(page).to have_content("Welcome: Han Solo")
```

Comment:
we can pass the name to the `set_omniauth` method so i think it's better to set that name in a variable and give it to omniauth and check here if he logged in